src/dmsc.d

```diff
@@ -113,7 +113,7 @@ void backend_init()
         params.symdebug,
         params.alwaysframe,
         params.stackstomp,
-        params.cpu == CPU.avx
+        params.cpu >= CPU.avx
```

Comment:
They're ordered so each successive one is a superset of the previous. Intel never removes instructions because of backwards compatibility, so the comparison is correct and robust in the case of newer instruction sets.